phalcon/config.zep

```diff
@@ -79,6 +81,58 @@ class Config implements \ArrayAccess, \Countable
 	}
 
 	/**
+	 * Sets the default path delimiter
+	 */
+	public function setPathDelimiter(string! delimiter) -> <Config>
+	{
+		let self::_delimiter = delimiter;
+		return this;
+	}
+
+	/**
+	 * Returns a value from current config using a dot separated path.
+	 *
+	 *<code>
+	 * echo $config->get("unknown.path", "default", ".");
+	 *</code>
+	 */
+	public function path(string path, var defaultValue = null, var delimiter = null) -> var
+	{
+		var key, keys, config;
+
+		if isset this->{path} {
+			return this->{path};
+		}
+
+		if empty delimiter {
+			let delimiter = self::_delimiter;
+		}
+
+		let keys = explode(delimiter, path),
```

Comment:
Actually when i tried to reproduce seg fault i have problem on this line if there is no delimiter argument provided.